GliaWidgets/Component/AttachmentList/Item/AtttachmentSourceItemKind.swift

```diff
@@ -0,0 +1,11 @@
+/// Kind of an item shown in the list view. Used in chat's attachment popover menu.
+public enum AtttachmentSourceItemKind {
+    /// Photo Library item.
+    case photoLibrary
+
+    /// Take Photo item.
+    case takePhoto
+
+    /// Browse item.
```

Comment:
Yes, precisely. Chat supports not only images but some other file types as well, which can be selected from the Files app.

Reply:
Maybe then something like:
```
/// Browse item. Allows sending various types of files available in the device's file system. The exact types of files allowed, depend on the site's settings.
```

I added the note about site's settings because I suspect the file types might be limited in mobile SDK in the same way as in the rest of the Glia platform, e.g. I give screenshot of the Glia Hub UI.
If the file types are limited in some other way, please correct me :)
<img width="1243" alt="file_types" src="https://user-images.githubusercontent.com/24471516/122679272-dfaed580-d1f2-11eb-9373-f4b8766f21e9.png">